zbus/src/connection/builder.rs

```diff
@@ -266,6 +274,10 @@ impl<'a> Builder<'a> {
     /// interfaces available immediately after the connection is established. Typically, this is
     /// exactly what you'd want. Also in contrast to [`zbus::ObjectServer::at`], this method will
     /// replace any previously added interface with the same name at the same path.
+    ///
+    /// # Errors
+    ///
+    /// Returns an error if and only if calling `ObjectPath::try_from(path)` fails.
```

Comment:
`try_from` is an implementation detail, one that's obvious from the generic constraint. I'd write "if `path` is not a valid object path".

Reply:
Done.